keyboards/adukif_75/config.h

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2020 bt66tech
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 15
+
+#define MATRIX_COL_PINS { C13, A6, A7, B0, B1, B12, B13, B14, B15, A8, A9, A10, B5, B8, B9 }
+#define MATRIX_ROW_PINS { A5, A4, A3, A2, A1, A0 }
+#define DIODE_DIRECTION ROW2COL
```

Comment:
You can move this (and most other definitions in this file) to `info.json`.

Reply:
A lot of this can be generated by running `qmk info -f json -kb (name)`.